### PR TITLE
Fix Internet Explorer compatibility

### PR DESCRIPTION
**Pull request by alexweber**

### CODE REVIEW

File: src/settings.service.js [hunk 1]
```diff
@@ -554,23 +554,29 @@ plugin.service('wgnConfigSettings', ['$q', 'wgnConfigInputs', function ($q, conf
...
-		 * Based on https://gist.github.com/eek/9c4887e80b3ede05c0e39fee4dce3747
+		 * Based on https://gist.github.com/codeguy/6684588
 		 */
 		function slugify(text) {
-			var slug = text.toString().trim()
-				.normalize('NFD') 				 // separate accent from letter
-				.replace(/[\u0300-\u036f]/g, '') // remove all separated accents
-				.replace(/(\-|\_)/g, '')        // remove hipens and underscores
-				.replace(/\s+/g, '')            // remove spaces
-				.replace(/&/g, '-and-')          // replace & with 'and'
```
Comment (tehpsalmist) [inline src/settings.service.js]: we won't handle ampersands anymore?

Comment (alexweber) [reply src/settings.service.js]: better fix ensues...

**Comment by tehpsalmist** [comment]
looking at srv.id, I don't think that function is used anywhere. We should delete it if so.

**Comment by tehpsalmist** [comment]
Eventually we need to actually test this out in IE lol...

**Comment by alexweber** [comment]
> looking at srv.id, I don't think that function is used anywhere. We should delete it if so.

Found this: https://github.com/ZengineHQ/zn-frontend-config/wiki/Ids

> Eventually we need to actually test this out in IE lol...

Agreed :)

**Comment by JoeBello** [comment]
Looks good in IE11

**Pull request merged**